src/test/compile-fail/feature-gate-negate-unsigned.rs

```diff
@@ -17,7 +17,8 @@ impl std::ops::Neg for S {
 }
 
 const _MAX: usize = -1;
-//~^ ERROR unary negation of unsigned integer
+//~^ WARN unary negation of unsigned integer
+//~| ERROR unary negation of unsigned integer
```

Comment:
Well... we can turn the lint into a `deny`-by-default lint and then we'd never reach the error